install_config/redeploying_certificates.adoc

```diff
@@ -385,9 +385,17 @@ file:
 
 ----
 $ ansible-playbook -i <inventory_file> \
-    /usr/share/ansible/openshift-ansible/playbooks/openshift-master/redeploy-certificates.yml
+    /usr/share/ansible/openshift-ansible/playbooks/openshift-master/redeploy-master-certificates.yml
```

Comment:
The playbook name should not be changed. The previous one is correct for ocp-3.9.
What we may need to improve is "*_redeploy-master-certificates.yml_* playbook" in line380, for more specifically, we should name it as "*_master redeploy-certificates.yml_* playbook".
So is in line 402 and line 421, which should be:
The *_etcd redeploy-certificates.yml_* playbook
The *_node redeploy-certificates.yml_* playbook